### PR TITLE
[Elm] Exercise 2-b

### PR DESCRIPTION
**Pull request by mpizenberg**
This exercise extends the pizza parsing exercise 1-b by adding a list of ingredients. I have done two implementations for exercise 2-b, one based on string manipulations, the other on Elm parser library. I'd like to add a third based on [regexes](https://package.elm-lang.org/packages/elm/regex/latest/), but I've very little regex experience.

### CODE REVIEW

File: exercises/elm-2-b/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+This is Part 1 of our research into how people write Elm differently.
+This exercise should take 10-15 minutes for a programmer who is proficient in Elm.
+There is no right or wrong way to approach it - just do what feels the most natural to you.
+Feel free to use books, Google or Stack Overflow,
+just like you would if you were programming normally,
+but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Well done on the printed menu analysis!
```
Comment (ErikSchierboom) [inline exercises/elm-2-b/README.md]: For the instruction, we usually repeat what the previous instructions were, as these are the instructions that are implemented by the starting code. For an example, see https://github.com/exercism/research_experiment_1/tree/master/exercises/elm-2-a#instructions.

**Comment by mpizenberg** [comment]
Noted! I'll fix this on Thursday

**Comment by ErikSchierboom** [comment]
Brilliant!

File: exercises/elm-2-b/README.md (hunk 1 above)
Comment (mpizenberg) [reply exercises/elm-2-b/README.md]: Ok, I've changed this in 51682d5

**Comment by mpizenberg** [comment]
I'm left with the regex implementation for the third start. I'll try to do this asap.

**Comment by mpizenberg** [comment]
The regex start is done in 6b471a4. Now waiting for your approval. Do I have to modify other files outside of `exercises/elm-2-b/`?

**Comment by mpizenberg** [comment]
Added. I tried to not change the last line of the file, but whatever the text editor I use, I can't seem to manage to not change that last line of `config.json` file.

**Comment by ErikSchierboom** [comment]
Thanks @mpizenberg! 🎉

**Pull request merged**